.gitignore

```diff
@@ -4,3 +4,4 @@ coredns.exe
 build/
 release/
 vendor/
+.idea/
```

Comment:
Please don't include your local editor ignores in the repo. Use [`git config --global core.excludesFile`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
good tip! removed